extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -228,8 +228,9 @@ export class RealTimeConfigManager {
    * custom URL.
    */
   modifyRtcConfigForConsentStateSettings() {
-    if (!(this.consentState_ == CONSENT_POLICY_STATE.INSUFFICIENT ||
-          this.consentState_ == CONSENT_POLICY_STATE.UNKNOWN)) {
+    if (this.consentState_ == undefined ||
```

Comment:
I believe we pass null around more than undefined (see https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/amp-a4a.js#L709) but I suppose == accomplishes this.  Do we have a test where null is explicitly passed (I believe we have tests where the value given is undefined).

Reply:
done